proj/proj.mk

```diff
@@ -366,11 +366,15 @@ else
 load: $(CFU_VERILOG) $(SOFTWARE_BIN)
 	$(SIM_MK) run
 
+run: $(SOFTWARE_BIN)
+	@echo Running automated test in Verilator simulation
+	$(BUILD_DIR)/interact.expect s $(RUN_MENU_ITEMS) |& tee $(UNITTEST_LOG)
```

Comment:
Actually, wait Tim. I think you would want to update the `tee` file to not be `UNITTEST_LOG`.